macros/src/lib.rs

```diff
@@ -209,7 +211,60 @@ trait AttributedItem {
     fn set_item_attributes(&mut self, attrs: Vec<Attribute>);
 }
 
-impl AttributedItem for Item {
+impl AttributedThing for Expr {
```

Comment:
btw, this is one of the things `sin` would make easier. Sooo many methods that should be provided by default but aren't